### PR TITLE
MGDOBR-993: Use official SDK - APIs

### PR DESCRIPTION
**Pull request by jomarko**
For more details see: https://issues.redhat.com/browse/MGDOBR-993

### CODE REVIEW

**Comment by manstis** [comment]
I had just forwarded Wojciech's email to @kelvah and @vpellegrino.

Thank-you for taking the lead :smiley:

**Comment by jomarko** [comment]
@manstis @vpellegrino @kelvah please notice this is a draft, more for demonstrating issues, that still needs to be improved on AppSRE side probably or in our sandbox openapi yaml.

**Review by manstis — APPROVED** [review]
The `Error`/`ModelError` remains.. but at least this moves to the _official_ SDK.

File: .github/workflows/openapi-sdk-compatibility-check.yml [hunk 1]
```diff
@@ -1,18 +1,9 @@
-name: "Smart Events UI :: SDK <-> Openapi compatibility check"
+name: "Smart Events :: SDK :: Compatibility check :: (MGDOBR-868)"
 on: [pull_request, push]
```
Comment (danielezonca) [inline .github/workflows/openapi-sdk-compatibility-check.yml:2]: Is it useful to keep it enabled until MGDOBR-868 is fixed?

Comment (jomarko) [reply .github/workflows/openapi-sdk-compatibility-check.yml]: @danielezonca thank you for your comment. I removed the old compatibility check. It needs to be rewritten as part of MGDOBR-868.

**Review by vpellegrino — APPROVED** [review]
LGTM, thanks @jomarko

**Review by kelvah — APPROVED** [review]
LGTM. Thanks @jomarko :ok_hand: 

Couple of minor observations.

I guess we can get rid of [`openapi/generated/.openapi-generator-ignore`](https://github.com/jomarko/sandbox-ui/blob/MGDOBR-843/openapi/generated/.openapi-generator-ignore)

Also this two scripts can be safely removed:
https://github.com/jomarko/sandbox-ui/blob/38da7b06909425e2b79f5058abba43ee3bd82ca3/package.json#L24-L25

And, final question, is this already the latest version of the SDK? It doesn't seem to include the changes introduced by [#1033](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1033)

**Comment by jomarko** [comment]
@kelvah thank you, changes applied.

Yes we are using latest relase https://github.com/redhat-developer/app-services-sdk-js/releases

You are correct the modiefiedAt is not available in the latets official RHOSE TS SDK, however I prefer to ask for next SDK release and use it here as another task/PR.

**Pull request merged**